agent.go

```diff
@@ -215,6 +228,41 @@ func NewAgent(config *AgentConfig) (*Agent, error) {
 	}
 	a.haveStarted.Store(false)
 
+	a.maxBindingRequests = config.MaxBindingRequests
```

Comment:
Fantastic work!

Would you mind moving the defaultValues into `const` values?

Reply:
done!